modules/biobank/php/optionsendpoint.class.inc

```diff
@@ -138,18 +138,25 @@ class OptionsEndpoint extends \NDB_Page implements RequestHandlerInterface
         // TODO: This should eventually be replaced by a call directly to a
         // Candidate endpoint or Candidate controller that will be able to
         // provide Candidate Objects.
-        $query      = "SELECT c.CandID as id,
-                            PSCID as pscid,
-                            Sex as sex,
-                            GROUP_CONCAT(DISTINCT DiagnosisID) as diagnosisIds,
-                            GROUP_CONCAT(DISTINCT s.ID) as sessionIds
-                       FROM candidate c
-                            LEFT JOIN session s USING (CandID)
-                            LEFT JOIN candidate_diagnosis_rel USING (CandID)
-                       WHERE s.CenterID IN ($userCenters)
-                            AND s.ProjectID IN ($userProjects)
-                       GROUP BY
-                          CandID";
+        $query      = "
+            SELECT
+                c.CandID as id,
+                PSCID as pscid,
+                Sex as sex,
+                GROUP_CONCAT(DISTINCT DxEvolutionID) as diagnosisIds,
+                GROUP_CONCAT(DISTINCT s.ID) as sessionIds
+            FROM
+                candidate c
+                LEFT JOIN session s
+                  ON s.CandidateID=c.CandID
```

Comment:
I dont think thats correct.

@jeffersoncasimir correct me if I'm wrong but this should be 
```suggestion
                  ON s.CandidateID=c.ID
```

While row 143 stays as is!

Reply:
Got it, I changed any join to use c.ID instead of c.CandID.